app/src/main/java/org/gnucash/android/ui/settings/BackupPreferenceFragment.java

```diff
@@ -422,12 +437,32 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 			case REQUEST_RESOLVE_CONNECTION:
 				if (resultCode == Activity.RESULT_OK) {
 					mGoogleApiClient.connect();
-					Preference pref = findPreference(getString(R.string.key_dropbox_sync));
+					Preference pref = findPreference(getString(R.string.key_google_drive_sync));
 					if (pref == null) //if we are in a preference header fragment, this may return null
 						break;
-					toggleDropboxPreference(pref);
+					toggleGoogleDrivePreference(pref);
 				}
 				break;
 		}
 	}
+
+	@Override
+	public void onGoogleDriveFolderCreated(String folderId) {
```

Comment:
Please, if it's not used, remove it. Now it just adds noise. It can be easily added later, if it's really needed.